packages/data-context/src/sources/migration/codegen.ts

```diff
@@ -447,6 +459,61 @@ export function getSpecPattern (cfg: LegacyCypressConfigJson, testType: TestingT
   return specPattern
 }
 
+function formatWithBundledBabel (config: string) {
+  const ast = parse(config)
+
+  let { code } = generate(ast, {}, config)
+  // By default babel generates imports like this:
+  // const {
+  //   defineConfig
+  // } = require('cypress');
+  // So we replace them with a one-liner, since we know this will never
+  // be more than one import.
+  //
+  // Babel also adds empty lines, for example:
+  //
+  // export default defineConfig({
+  //   component: {
+  //   },
+  //               <===== empty line
+  //   e2e: {
+  //
+  //   }
+  // })
+  // Which we don't want, so we change those to single carriage returns.
+  const replacers = [
+    {
+      from: dedent`
+      const {
+        defineConfig
+      } = require('cypress');`,
+      to: dedent`
+      const { defineConfig } = require('cypress');`,
```

Comment:
```suggestion
      from: dedent`
        const {
          defineConfig
        } = require('cypress');
      `,
      to: dedent`
        const { defineConfig } = require('cypress');
      `,
```

This matches the style of other delineators, like {} and [].